agot-bg-game-server/src/common/ingame-game-state/game-data-structure/house-card/AeronDamphairDwDHouseCardAbility.ts

```diff
@@ -3,10 +3,16 @@ import HouseCard from "./HouseCard";
 import House from "../House";
 import BeforeCombatHouseCardAbilitiesGameState from "../../action-game-state/resolve-march-order-game-state/combat-game-state/before-combat-house-card-abilities-game-state/BeforeCombatHouseCardAbilitiesGameState";
 import AeronDamphairDwDAbilityGameState from "../../action-game-state/resolve-march-order-game-state/combat-game-state/before-combat-house-card-abilities-game-state/aeron-damphair-dwd-ability-game-state/AeronDamphairDwDAbilityGameState";
+import CombatGameState from "../../action-game-state/resolve-march-order-game-state/combat-game-state/CombatGameState";
 
 export default class AeronDamphairDwDHouseCardAbility extends HouseCardAbility {
     beforeCombatResolution(beforeCombat: BeforeCombatHouseCardAbilitiesGameState, house: House, _houseCard: HouseCard): void {
-        beforeCombat.childGameState.setChildGameState(new AeronDamphairDwDAbilityGameState(beforeCombat.childGameState))
-            .firstStart(house);
+        const aeronDamphairDwDAbilityGameState = new AeronDamphairDwDAbilityGameState(beforeCombat.childGameState);
+        this.abilityGameState = aeronDamphairDwDAbilityGameState;
```

Comment:
@Longwelwind I guess this is the reason... Clients don’t know the ability received a game state now...